utils/AppImage.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+echo "*****************************************"
+echo "Building Kaidan AppImage. This may take a long time"
+echo "*****************************************"
+
+# Download linuxdeployqt
+if ! [ -f linuxdeployqt-continuous-x86_64.AppImage ]; then
+    wget https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage
+    chmod +x linuxdeployqt-continuous-x86_64.AppImage
+fi
+
+# configure buildsystem and environment for AppDir installation
+mkdir ../build
```

Comment:
only create, if non-existant?

Reply:
Yeah I could do that but it would work anyway, just throw a warning. As this code will land on a  throw-away container on the CI most likely, it would not be that important.

Reply:
Not that important, but it's cleaner.